Makefile

```diff
@@ -43,29 +43,26 @@ deb:
 update-version:
 	$(MAVEN_BINARY) versions:set -DnewVersion=$(new-version) -DgenerateBackupPoms=false
 
-package-jdk8:
-	$(MAVEN_BINARY) clean package -Djdk.version=1.8
+package-jdk11:
+	$(MAVEN_BINARY) clean package -Djdk.version=11
```

Comment:
Passing `jdk.version` from environment variable might help reduce duplication of targets.

Reply:
yeah thats good idea, i was just lazy to do that before